config/config.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2019 ChainSafe Systems (ON) Corp.
+// This file is part of gossamer.
+//
+// The gossamer library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The gossamer library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the gossamer library. If not, see <http://www.gnu.org/licenses/>.
+
+package cfg
+
+import (
+	"github.com/ChainSafe/gossamer/p2p"
+	"github.com/ChainSafe/gossamer/polkadb"
+)
+// Config is a collection of configurations throughout the system
+type Config struct {
+	ServiceConfig *p2p.ServiceConfig
+	BadgerDB      polkadb.BadgerDB
```

Comment:
I agree, especially once we implement more configuration options for the badgerDB (eg. cachelimits etc). I will add a badgerDB config struct but at the current time it will only house the Datadir.